src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Webservice/index.html.twig

```diff
@@ -52,6 +52,36 @@
     {% endif %}
   {% endblock %}
 
+  {% block webservice_api_status %}
+    {% set devdocUrl = 'https://devdocs.prestashop.com/1.7/development/webservice/' %}
```

Comment:
should we make this URL configurable to ease updates?

Reply:
I don't think so. Some links are used in twig or smarty directly and others in PHP.